app/__init__.py

```diff
@@ -15,20 +15,26 @@ def create_app(test_config=None):
     app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 
     if test_config is None:
+            # uncomment when accesing render database:
+        # app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get("RENDER_DATABASE_URI")
+            # optional: after all test passing automate app to point to correct database
         app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
             "SQLALCHEMY_DATABASE_URI")
     else:
         app.config["TESTING"] = True
         app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
             "SQLALCHEMY_TEST_DATABASE_URI")
 
-    # Import models here for Alembic setup
     from app.models.task import Task
     from app.models.goal import Goal
-
+    
     db.init_app(app)
     migrate.init_app(app, db)
```

Comment:
Great job! We need to pass our instance of our app & the instance of our SQLALchemy db to connect the db and migrate to our Flask app